heareval/score.py

```diff
@@ -176,10 +205,18 @@ def __call__(self, predictions: Dict, targets: Dict, **kwargs):
                 estimated_event_list=estimated_event_list.filter(filename=filename),
             )
 
-        # This (and segment_based_scores) return a pretty large selection of scores.
-        overall_scores = scores.results_overall_metrics()
-        # Keep the specific score we want
-        return overall_scores[self.score][self.score]
+        # results_overall_metrics return a pretty large nested selection of scores,
+        # with dicts of scores keyed on the type of scores, like f_measure, error_rate,
+        # accuracy
+        nested_overall_scores: Dict[
+            str, Dict[str, float]
+        ] = scores.results_overall_metrics()
+        # Open up nested overall scores
+        overall_scores: Dict[str, Union[float, int]] = dict(
```

Comment:
Are you sure we want dict? Can't we keep the scores in the exact order of `scores` param? Seems safer that way.

Reply:
we get the dict. The final return is in the order of the scores param